erts/emulator/beam/beam_file.c

```diff
@@ -740,6 +749,7 @@ beamfile_read(const byte *data, size_t size, BeamFile *beam) {
         MakeIffId('C', 'I', 'n', 'f'), /* 8 */
         MakeIffId('L', 'i', 'n', 'e'), /* 9 */
         MakeIffId('A', 't', 'U', '8'), /* 10 */
+	MakeIffId('L', 'o', 'c', 'T'), /* 11 */
```

Comment:
Nitpick: Nowadays we intend with space, not tabs.

Reply:
Fixed.